designs/administrator/olm/catalogsource-ui-enhancements/index.md

```diff
@@ -0,0 +1,58 @@
+---
+parent: Administrator
+version: 4.7
+---
+
+# Catalog Source UI Enhancements
+
+Enabling and disabling default catalog sources (which appear in the OperatorHub as Provider Types) is now available from the Details tab of the OperatorHub settings resource.
+
+Users have been able to configure custom curated catalog sources as of OpenShift 4.4 with metadata like “Display Name”, “URL of Image Registry”, and the “Polling Interval.” Those custom sources now have their full details conveyed in the web console.
+
+## OperatorHub settings
+
+![OperatorHub settings](img/1-1.png)
+- The user opens the OperatorHub settings page
+
+![OperatorHub settings](img/1-2.png)
+- The user can now enable/disable default sources from the OperatorHub Details tab
+
+![OperatorHub settings](img/1-3.png)
+- A modal is opened that reflects the enabled/disabled state for the 4 default sources using switches
+
+![OperatorHub settings](img/1-4.png)
+- If 1 or more default sources are disabled, an inline alert appears reflecting ramifications
+
+![OperatorHub settings](img/1-5.png)
+- The source’s status is updated
+
+![OperatorHub settings](img/1-6.png)
+- The Sources tab list view now includes 2 new columns for the Catalog Sources: **Status** and **Registry Poll Interval**
+- The user opens the action menu(s)
+
+![OperatorHub settings](img/1-7.png)
```

Comment:
Small nit – it looks like where you added columns in the previous screen you have text here that doesn't match those column headers